greenhouse/assets/logs/greenhouse.yaml

```diff
@@ -0,0 +1,78 @@
+id: greenhouse
+metric_id: greenhouse
+backend_only: false
+facets:
+  - groups:
+      - Web Access
+    name: Client IP
+    path: network.client.ip
+    source: log
+  - groups:
+      - Geoip
+    name: City Name
+    path: network.client.geoip.city.name
+    source: log
+  - groups:
+      - Geoip
+    name: Continent Code
+    path: network.client.geoip.continent.code
+    source: log
+  - groups:
+      - Geoip
+    name: Continent Name
+    path: network.client.geoip.continent.name
+    source: log
+  - groups:
+      - Geoip
+    name: Country ISO Code
+    path: network.client.geoip.country.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Country Name
+    path: network.client.geoip.country.name
+    source: log
+  - groups:
+      - Geoip
+    name: Subdivision ISO Code
+    path: network.client.geoip.subdivision.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Subdivision Name
+    path: network.client.geoip.subdivision.name
+    source: log
+pipeline:
+  type: pipeline
+  name: greenhouse
+  enabled: true
+  filter:
+    query: "source:greenhouse"
+  processors:
+    - type: date-remapper
+      name: Define `event_time` as the official date of the log
+      enabled: true
+      sources:
+        - event_time
+    - type: service-remapper
```

Comment:
You have a service remapper but no service in your log sample

Reply:
Done 👍